Feature branch

torobucci opened the pull request:
**#Hello World project requirements**
The feature branch makes the following changes as required in the hello world project

- Adds and modifies the .github folder and add linters.yml to the workflow folder. There confirms there is no typo errors.
- Modifies the README.md file to be more informative as the Microverse README.md file
- Adds and modifies the index.html  to have proper basic html code and include a `h1`element
- Adds and modifies the style.css file. At first the file wasn't named properly
- Adds the .gitignore file and includes the test.md and node_modules to be ignored by git

torobucci committed:
Set up a linter for Html and Css

torobucci committed:
Added index.html file with a 'Hello Microverse' h1 element

torobucci committed:
Added .css file with header styles

torobucci committed:
Added test.md file

torobucci committed:
Cutomized the readme.md file to readme.md microverse template

torobucci committed:
Added test.md file to the .gitignore file

torobucci committed:
Added package.json module

Abenezer-Tilahun commented on index.html:
```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse</h1>
```
- [ ] Kindly I recommend you to follow the best practices of HTML5 Syntax, For More information you can refer to this [document](https://www.w3schools.in/html5/syntax) Just Like this : 
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="stylesheet" type="text/css" href="Style.css">
        <title></title>
        </head>
        <body>
            <h1 class=""></h1>
        </body>
        </html>
```

- [ ]  As stated the project requirement Kindly make sure that A .css file that includes `one class` with styles for your header (make it in your favorite color).
- ![Capture](https://user-images.githubusercontent.com/93598189/211501647-d22fcab1-eabd-46f6-84a6-f12672abe6f9.PNG)

Abenezer-Tilahun replied:
Fixed.

Abenezer-Tilahun commented on .css:
```diff
@@ -0,0 +1,4 @@
+header{
+    background-color: gray;
+    color: white;
+}
```
- [ ] For semantics correct and best practices of Style-sheet , Kindly I recommend you to rename your .`css `file which helps to link Html file Just like this : 
`<link rel="stylesheet" href="Style.css">`

Abenezer-Tilahun replied:
Fixed.

Abenezer-Tilahun commented on test.md:
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```
- [ ] As stated the project requirement Kindly make sure that `test.md` file is actually ignored by git and is not present in your GitHub repository.

![Capture](https://user-images.githubusercontent.com/93598189/211783152-2ab17b7d-bec3-4fad-a8f0-c917f7ef9fbb.PNG)

Abenezer-Tilahun replied:
Fixed.

Abenezer-Tilahun commented on README.md:
```diff
@@ -1 +1,8 @@
+  Css
+  ## Author
+  GitHub:@torobucii
+  
```
- [ ] Kindly I recommend you to update your README file with the new [README template](https://github.com/microverseinc/readme-template)  , **All main headers** from the [README template](https://github.com/microverseinc/readme-template) should be present in a yours repo and Please Make sure the you has added the **section Future Features**, including 1-3 potential features to be added..

Abenezer-Tilahun replied:
Fixed.

Abenezer-Tilahun commented on .gitignore:
```diff
@@ -0,0 +1 @@
+test.md
```
- [ ] As stated the project requirement Kindly make sure that Added `node_modules` to `gitignore` file, when you Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.

Abenezer-Tilahun replied:
Fixed.

Abenezer-Tilahun commented on .github/workflows:
```diff
@@ -0,0 +1,56 @@
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@7.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
- [x] Kindly Make sure that Linters set up correctly for this repository.Linter result should be green, this means that the last commit have a green check mark (✅) or a summary of the passed checks is in the PR. Your [linters.yml](https://github.com/Kevin-Mena/helloworldmicroverse/blob/html/css/.github/workflow/linters.yml) file is empty Kindly update with correct [one](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml)
- Kindly follow the steps for configure the [linters correctly](https://github.com/microverseinc/linters-config/tree/master/html-css) and follow the correct flow of[.github](https://github.com/microverseinc/linters-config/tree/master/html-css/.github)/[workflows](https://github.com/microverseinc/linters-config/tree/master/html-css/.github/workflows)/linters.yml

Abenezer-Tilahun replied:
Fixed.

Abenezer-Tilahun reviewed (changes requested):
Hi @torobucci  , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ You used correct Git Hub-Flow.
✔️ Good commit messages

![almost](https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif)

## Required Changes ♻️

- [ ]  Kindly make sure that Pull Request  have a **descriptive title** telling you what changes are inside.

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

torobucci committed:
Modified index.html to include basic html tags

torobucci committed:
renamed .css file to style.css

torobucci committed:
Delete .css

torobucci committed:
added linters.yml file to the .github/workflows

torobucci committed:
added .github folder

torobucci committed:
Added .hintrc .stylelintric.json package-loc.json files

torobucci committed:
deleted test.md file

test.md file had already been commited to git hence unable to be ignored by git

torobucci committed:
deleted test.md

torobucci committed:
deleted test.md file

torobucci committed:
Updated README.md file 

To be similar to the Microverse README.md template

torobucci committed:
Update README.md

Edited the template header

torobucci committed:
Update README.md

Abenezer-Tilahun commented on index.html:
```diff
@@ -0,0 +1,13 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```
- [Optional  ] As stated the project requirement Kindly make sure that A .css file that includes `one class` with styles for your header (make it in your favorite color).
- ![Capture](https://user-images.githubusercontent.com/93598189/211501647-d22fcab1-eabd-46f6-84a6-f12672abe6f9.PNG)

Abenezer-Tilahun reviewed (approved):
Hi @torobucci 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
